src/vit-servicing-station/vit-servicing-station-cli/src/api_token.rs

```diff
@@ -122,13 +122,16 @@ impl ApiTokenCmd {
         }
     }
 
-    fn handle_api_token_add_whith_db_backup(
+    fn handle_api_token_add_with_db_backup(
         tokens: &Option<Vec<String>>,
         db_url: &str,
     ) -> Result<(), Error> {
-        let backup_file = backup_db_file(db_url)?;
         if let Err(e) = Self::handle_api_token_add(tokens, db_url) {
-            restore_db_file(backup_file, db_url)?;
+            if !db_url.starts_with("postgres://") {
```

Comment:
I'm not sure why we backup the DB file. Restoring the Postgres DB wouldn't be as simples as copying file contents so I kept this only for SQLite.